romancal/pipeline/mosaic_pipeline.py

```diff
@@ -77,7 +80,8 @@ def process(self, input):
             # check to see if the product name contains a skycell name & if true get the skycell record
             product_name = input.asn_table["products"][0]["name"]
             try:
-                skycell_name = product_name.split("_")[3]
+                skycell_name = product_name.split("_")[1]
+                skycell_name = input.asn_table["target"]
```

Comment:
At present the product_name-based skycell_name is immediately overwritten by the target-based one?  Probably just delete line 83 if we have updated asn files?

Reply:
And now you know why it is a draft.